packages/flutter/lib/src/services/raw_keyboard_android.dart

```diff
@@ -216,40 +216,27 @@ class RawKeyEventDataAndroid extends RawKeyEventData {
     if (metaState & anyMask == 0) {
       return false;
     }
-    switch (side) {
-      case KeyboardSide.any:
-        return true;
-      case KeyboardSide.all:
-        return metaState & leftMask != 0 && metaState & rightMask != 0;
-      case KeyboardSide.left:
-        return metaState & leftMask != 0;
-      case KeyboardSide.right:
-        return metaState & rightMask != 0;
-    }
+    return switch (side) {
+      KeyboardSide.any   => true,
+      KeyboardSide.all   => metaState & leftMask != 0 && metaState & rightMask != 0,
```

Comment:
Thoughts on this? I don't feel strongly:

```suggestion
      KeyboardSide.all   => (metaState & leftMask != 0) && (metaState & rightMask != 0),
```

Reply:
Works for me!